$NetBSD: patch-ext_wayland_meson.build,v 1.1 2021/11/15 22:01:18 wiz Exp $

Exit the Wayland subdirectory if the option is disabled.

This is necessary to build the GStreamer libraries separately
from the extension modules, since use_wayland (which is checked later)
will be undefined in an extension build and cause an evaluation error.

--- ext/wayland/meson.build.orig	2021-09-08 19:03:40.875254400 +0000
+++ ext/wayland/meson.build
@@ -1,3 +1,8 @@
+# Don't do any dependency checks if disabled
+if get_option('wayland').disabled()
+  subdir_done()
+endif
+
 wl_sources = [
     'gstwaylandsink.c',
     'wlshmallocator.c',
